### PR TITLE
Add licences to example files that don't have one

### PR DESCRIPTION
**Pull request by vtomole**
Fixes https://github.com/quantumlib/Cirq/issues/912

### CODE REVIEW

**Comment by vtomole** [comment]
Sure. The thing is that some files have notices and some don't. @Strilanc should it be all or none?

**Comment by Strilanc** [comment]
None.

File: examples/README.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+The software in this directory (Cirq/examples/) is not included in the packaged library. Anyone is free to copy, modify, or publish this software.
```
Comment (Strilanc) [inline examples/README.rst]: "this example code" instead of "this software". And add "use" in the list of things they're allowed to do.

Comment (vtomole) [reply examples/README.rst]: Done.

**Pull request merged**